operator/hack/csv-generate.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+export GOROOT=$(go env GOROOT)
+
+#PREV="0.0.1"
+LATEST="0.0.1"
+
+if [[ -z "$CSV_VERSION" ]]; then
+  CSV_VERSION=$LATEST
+  REPLACES_CSV_VERSION=$PREV
+fi
+
+PACKAGE_NAME="observatorium-operator"
+
+PACKAGE_DIR="${CATALOG_DIR}/${PACKAGE_NAME}"
+CSV_DIR="${PACKAGE_DIR}/${CSV_VERSION}"
+CSV_FILE="$CSV_DIR/${PACKAGE_NAME}.v${CSV_VERSION}.clusterserviceversion.yaml"
+
+OUT_DIR="_output/olm-catalog"
+OUT_CSV_DIR="${OUT_DIR}/${PACKAGE_NAME}/${CSV_VERSION}"
+OUT_CSV_FILE="${OUT_CSV_DIR}/${PACKAGE_NAME}.v${CSV_VERSION}.clusterserviceversion.yaml"
+
+ANNOTATIONS_FILE="${DEPLOY_DIR}/csv/annotations.json"
+MAINTAINERS_FILE="${DEPLOY_DIR}/csv/maintainers.json"
+
+EXTRA_ANNOTATIONS=""
+MAINTAINERS=""
+
+if [ -n "$DESCRIPTION_FILE" ]; then
+	DESCRIPTION="-description-from=$DESCRIPTION_FILE"
+fi
+if [ -n "$MAINTAINERS_FILE" ]; then
+	MAINTAINERS="-maintainers-from=$MAINTAINERS_FILE"
+fi
+if [ -n "$ANNOTATIONS_FILE" ]; then
+	EXTRA_ANNOTATIONS="-annotations-from=$ANNOTATIONS_FILE"
+fi
+
+clean_package() {
```

Comment:
This function is only used once; why not inline it?

Reply:
could be helpful for future reusability, but we can indeed inline it for now.